src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[]{new Bulldozer(), new Excavator(), new Truck()};
+        for (Machine m: machines) {
```

Comment:
i'd rather give it a full name
```suggestion
        for (Machine machine: machines) {
```